src/components/ToDoList.vue

```diff
@@ -1,35 +1,49 @@
 <template>
-  <li v-for="(todo, index) in toDos" v-bind:key="todo.id">
-    <div class="p-6 bg-gray-200 rounded-lg border-radius">
-      <div class="color #000000">{{ todo.title }}</div>
-      <div class="color #000000">{{ todo.text }}</div>
-      <div class="color #000000">{{ todo.priority }}</div>
-      <div class="color #000000">{{ todo.created_at }}</div>
-      <div class="color #000000">{{ todo.status }}</div>
-    </div>
-
-    <button class="border-radius: 50%;" @click="removeToDo(index)">
-      delete
-    </button>
-  </li>
-  <div></div>
-
-  <img />
+  <div v-for="(todo, index) in toDos" v-bind:key="todo.id">
+    <ToDoItem

```

Comment:
Please refactor this code block: there are set two TodoItem component almost the same attributes. The only one difference is the toDoUpdated function. 
In this case please decide that prop only based on the condition. You don't need to duplicate the components.